packages/accounts/src/msca/client.ts

```diff
@@ -48,8 +74,7 @@ export function createMultiOwnerModularAccountClient<
     CustomTransport,
     Chain,
     MultiOwnerModularAccount<TSigner>,
-    SmartAccountClientActions<Chain, MultiOwnerModularAccount<TSigner>> &
-      MultiOwnerPluginActions<MultiOwnerModularAccount<TSigner>> &
+    MultiOwnerPluginActions<MultiOwnerModularAccount<TSigner>> &
```

Comment:
why dont we need `SmartAccountClientActions` on this generic anymore?

Reply:
see comment above